### PR TITLE
Ch 1703: Don't show report charts/tables when decisions hasn't been shown.

### PR DESCRIPTION
**Pull request by eshtadc**
Note: the two overview report function chunks are just moving the functions from the Lift report class down to the base class for use with both lift report types.

### CODE REVIEW

File: plugins/agent_types/AcquiaLiftAgent.inc [hunk 1]
```diff
@@ -2248,47 +2319,49 @@ class AcquiaLiftReport extends AcquiaLiftReportBase {
...
+            'conversion' => $choice['conversion'],
+          );
+          $option_numbers[$feature_string . '|' . $choice['raw_label']] = $choice['counter'];
+        }
       }
     }
 
     // Build the experiment report data
-    $report_data['experiment']['choices'] = $report_data['confidence']['features'][self::NO_FEATURES];
+    $report_data['experiment']['choices'] = isset($report_data['confidence']['features'][self::NO_FEATURES]) ? $report_data['confidence']['features'] : array();
```
Comment (katbailey) [inline plugins/agent_types/AcquiaLiftAgent.inc:2364]: Just checking that this is what's intended - should it not be
`isset($report_data['confidence']['features'][self::NO_FEATURES]) ? $report_data['confidence']['features'][self::NO_FEATURES] : array();`
?

File: tests/acquia_lift.test [hunk 2]
```diff
@@ -1480,7 +1480,6 @@ class AcquiaLiftWebTestAgentAdmin extends AcquiaLiftWebTestBase {
     $this->personalizedQueue->deleteQueue();
 
     $goals = personalize_goal_load_by_conditions(array('agent' => $agent_name));
-    debug($goals);
```
Comment (katbailey) [inline tests/acquia_lift.test:1483]: :)

File: plugins/agent_types/AcquiaLiftAgent.inc (hunk 1 above)
Comment (eshtadc) [inline plugins/agent_types/AcquiaLiftAgent.inc:2364]: See!  This is why I need your code reviews :-)

Comment (katbailey) [inline plugins/agent_types/AcquiaLiftAgent.inc:2364]: Oh yay - I was useful! :D

**Pull request closed without merging**